Source/Blazorise/Components/Highlighter/Highlighter.razor.cs

```diff
@@ -64,6 +93,11 @@ private static IEnumerable<string> GetFragments( string text, string highlighted
     /// </summary>
     [Parameter] public string HighlightedText { get; set; }
 
+    /// <summary>
+    /// Array of search terms to be highlighted.
+    /// </summary>
+    [Parameter] public List<string> HighlightedTexts { get; set; }
```

Comment:
I think it would be best to have it as an array

`public string[] HighlightedTexts`

That way we will force users to reasign HighlightedTexts parameter and we will not have problems with mutating of list and comparing reference types as we have in other component.

Reply:
They would be less likely to update an array as `myArray[1] = "something else";` I agree with you. The problem still occurs. 

I think the best way to say it in the documentation is that mutation is generally not supported for reference types and that users need to recreate a new list if they wish to change the parameter.